policies/project-director-election-process.md

```diff
@@ -0,0 +1,68 @@
+# Project Director Election Process
+
+## Goals
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria in the [role description document](TODO: link), focusing on shared power and new leadership rather than solely prioritizing the best candidate.
```

Comment:
"Shared power" might need some explanation as I don't think its obvious what this means. I would offer some wording, but I'm not sure I understand what is meant.